map_gui/src/render/mod.rs

```diff
@@ -50,10 +50,8 @@ pub trait Renderable {
     }
     // This outline is drawn over the base object to show that it's selected. It also represents
     // the boundaries for quadtrees. This isn't called often; don't worry about caching.
-    fn get_outline(&self, map: &Map) -> Polygon;
-    fn contains_pt(&self, pt: Pt2D, map: &Map) -> bool {
-        self.get_outline(map).contains_pt(pt)
-    }
+    fn get_outline(&self, map: &Map) -> Tessellation;
```

Comment:
Necessary because some buildings have holes. `Polygon::to_outline` returns a `Tessellation` now to handle that case. Returning a `Vec<Polygon>` (one for exterior, one for each hole) is another option, but it's a gross API. For the callers that want `Polygon` outlines for each individual ring, they can still do that themselves